homeassistant/const.py

```diff
@@ -767,8 +767,17 @@ class UnitOfMass(StrEnum):
 # Conductivity units
 CONDUCTIVITY: Final = "µS/cm"
 
+
 # Light units
+class UnitOfIlluminance(StrEnum):
+    """Illuminance units."""
+
+    LUX = "lx"
```

Comment:
There's only one unit. Are we planning to add more to this enum?

Reply:
No, I was not planning to add more units to this enum, but it is linked to `ILLUMINANCE` device class.

The current documentation states that we also allow lumen (lm) for the device class, but I think that shoud be removed from the documentation, and maybe add a specific `LUMINOUS_FLUX` device class at a later stage.

Reply:
I don't think it's worth it to change all units if there is only one item in the enum.

Reply:
I think that if we have a device class, and we specify which units are available under that device class, then I do think it makes sense to use an Enum.

Reply:
Why? What's better with a one item enum compared to a constant?

Reply:
This is not about better or not. That is a non-discussion, as both are constant; one lives in an enum, and the other doesn't.

This is about consistency. If we move all units to enums in the core, it would be odd to leave one out.

That said, we can leave this one out for now and maybe look at it again later. 🤷

Reply:
Yeah, if it's only about consistency I don't think it's worth it, unless it's the final one left. There are more single item units left as far as I know.

Reply:
That sounds like a good compromise.

@epenet Going to close this for now. Let's revisit this later.